src/ipld/formats/pb/mod.rs

```diff
@@ -96,41 +96,56 @@ impl Into<Ipld> for PbLink {
 }
 
 impl TryFrom<Ipld> for PbNode {
-    type Error = std::option::NoneError;
+    type Error = TryError;
 
     fn try_from(ipld: Ipld) -> Result<PbNode, Self::Error> {
         match ipld {
             Ipld::Object(mut map) => {
-                let links: Vec<Ipld> = map.remove("Links")?.try_into()?;
+                let links: Vec<Ipld> = match map.remove("Links") {
```

Comment:
wouldn't `.map_or(TryError)` be nicer than using `match` everywhere?

Reply:
The `.unwrap_or()` function doesn't return the `try_from` function, it tries to set an alternative `Ipld` value.

When I try to use `.unwrap_or(TryError)` here I get:
`src/ipld/formats/pb/mod.rs|104 col 70 error 308| mismatched types: expected enum 'ipld::ipld::Ipld', found struct 'error::TryError'`